spec/helpers/applications_helper_spec.rb

```diff
@@ -171,6 +171,18 @@
       end
     end
 
+    describe "applications_json_path" do
```

Comment:
This does the job, but it makes the [separate test phases hard](https://robots.thoughtbot.com/four-phase-test) to read.

Also using [BetterSpecs](http://betterspecs.org/#subject) advice about using 'subject' etc. something like this might be clearer:

```
    describe "applications_json_path" do
      let(:authority) { create(:authority, short_name: "marrickville", ) }
      let(:user) { build(:user, api_key: "ABCDE12345") }

      before { expect(helper).to receive(:current_user).and_return(user) }

      subject { helper.applications_json_path(authority) }

      it { is_expected.to eq("/authorities/marrickville/applications.js?key=ABCDE12345") }
    end
```

Note that it uses `build` not `create` for the user, to avoid the before_create callback.